src/data/contributors/AsifNasim.json

```diff
@@ -0,0 +1,10 @@
+{
+    "name": "Md Asif Nasim",
+    "skills": ["Java", "React", "JavaScript", "Django"],
+    "cause": "Extracting Wisdom From It",
+    "image": "Avatar3",
+    "github": "https://github.com/AsifNasim",
+    "twitter": "https://twitter.com/",
```

Comment:
Leave it blank if dont have twitter id else provide the full url. 
Actually these url provides the way to connect to people. :)

Reply:
Thanks a lot. I will make an account right away and make changes in it.